docs/_components/modal.md

```diff
@@ -59,14 +61,15 @@ Use the `.modal--danger` class for situations where we're performing destructive
 ```
 
 <div class="pulsar-example">
-    <div class="modal modal--danger modal__example show">
+    <div class="modal modal--danger modal__example show" id="myModal-3" role="dialog" aria-labelledby="dialog-title-3" aria-describedby="dialog-description-3">
         <div class="modal__dialog">
             <div class="modal__content">
                 <div class="modal__header">
-                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
-                    <h4 class="modal__title"><i class="icon-warning-sign"></i> You’re about to do something really, really bad</h4>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close dialog">×</button>
```

Comment:
I know this was already an issue before this PR, but I noticed some examples use `&times;` and some use `×`